core/sdk/src/test/java/com/mongodb/stitch/core/auth/internal/CoreStitchAuthUnitTests.java

```diff
@@ -17,14 +17,16 @@
 package com.mongodb.stitch.core.auth.internal;
```

Comment:
[task] can you also add tests where you:
* log back in to a logged out (but not removed) user
* log back in to a removed user